kitchen.yml

```diff
@@ -5,31 +5,23 @@ driver:
 
 provisioner:
   product_name: chef
-  product_version: <%= ENV['CHEF_VERSION'] || "current" %>
+  product_version: 15
```

Comment:
Why not "current" or 16?

Reply:
I agree, let's keep this as it was.

Reply:
I changed it because it didn't work.  I'll update it to use latest instead of current.  Thanks for the comments.